pkg/filesystem/windowsext/types.go

```diff
@@ -3,7 +3,13 @@
 
 package windowsext
```

Comment:
To be honest, I've never been a big fan of the existence of this package. These are all fairly generic definitions that could also be made part of `golang.org/x/sys/windows`. Would you by any chance be interested in submitting PRs against that package to see how willing they are to merge such changes?

Reply:
If you could merge as-is please that would help as the WinFSP work requires this. Though if you prefer I could add the necessary extra bits privately in bb-remote-execution. It just seemed nicer to put them here where the other types were.

I'll submit a PR for `golang.org/x/sys/windows` as well and if that gets merged I'll follow-up here and delete this file. But I'm guessing that will take a while as it would require a new go release.